example/ck_tile/35_batched_transpose/batched_transpose.cpp

```diff
@@ -0,0 +1,310 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#include <vector>
+#include <iostream>
+#include <numeric>
+#include <cassert>
+#include <cstdlib>
+#include <iostream>
+#include <time.h>
+#include <unordered_set>
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/reduce.hpp"
+#include "batched_transpose_api.hpp"
+#include "batched_transpose.hpp"
+
+#ifndef TEST_BATCHED_TRANSPOSE_VERBOSE
+#define TEST_BATCHED_TRANSPOSE_VERBOSE 1
+#endif
+
+template <typename T>
+void dump_host_tensor_4d(const ck_tile::HostTensor<T>& x)
```

Comment:
Please comment this out or put that to another file as a helper debug function. We could get a debug option in the argument input. WDYT?